doc/man7/Ed25519.pod

```diff
@@ -7,9 +7,9 @@ Ed25519 - EVP_PKEY Ed25519 support
 =head1 DESCRIPTION
 
 The B<Ed25519> EVP_PKEY implementation supports key generation, one shot
-digest sign and digest verify using PureEdDSA and B<Ed25519> (see RFC8032).
-It has associated private and public key formats compatible with
-draft-ietf-curdle-pkix-04.
+digest sign and digest verify using PureEdDSA and B<Ed25519> (see
+L<RFC8032|https://tools.ietf.org/html/rfc8032>). It has associated private
```

Comment:
There are about 150 RFC references in our manpages, and changing just this one to be an Link seems wrong.  Please turn edit to "RFC 8032"  Or fix all of the others :)